spec/spec_helper.rb

```diff
@@ -19,11 +19,23 @@
 # in ./support/ and its subdirectories.
 Dir[File.expand_path(File.join(__dir__, 'support/**/*.rb'))].each { |f| require f }
 
+class TrollopEducateSpecError < StandardError; end
+class TrollopDieSpecError < StandardError; end
+
 RSpec.configure do |config|
   config.after(:each) do
     Module.clear_all_cache_with_timeout if Module.respond_to?(:clear_all_cache_with_timeout)
   end
 
+  config.before(:each) do
```

Comment:
No, this has to be per-test. The error I get from rspec if I do `before(:suite)` or `before(:all)` is:

>The use of doubles or partial doubles from rspec-mocks outside of the per-test lifecycle is not supported.